tests/phpunit/tests/test-wpml-cornerstone-translatable-nodes.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * Test_WPML_Cornerstone_Translatable_Nodes test file.
+ *
+ * @package wpml-page-builders-cornerstone
+ */
+
+/**
+ * Class Test_WPML_Cornerstone_Translatable_Nodes
+ *
+ * @group page-builders
+ * @group cornerstone
+ * @group wpmltm-3081
+ */
+class Test_WPML_Cornerstone_Translatable_Nodes extends OTGS_TestCase {
+
+	/**
+	 * Test get method.
+	 *
+	 * @dataProvider node_data_provider
+	 *
+	 * @param string $type                 Node type.
+	 * @param string $field                Node field.
+	 * @param string $expected_title       Expected title.
+	 * @param string $expected_editor_type Expected editor type.
+	 */
+	public function test_get( $type, $field, $expected_title, $expected_editor_type ) {
```

Comment:
@kagg-design why you don't use the current convention of writing the test method as a sentence? `it_does_that`

Reply:
@andreasciamanna because it saves one line in phpDoc

    * @test

Function which name starts with `test` are considered as tests by default. We cann add any sentence to it, like `test_get_with_invalid_node()`.

We have tests when function are named in tihs way.

Anyway, if `it_does` is considered as a company standard, I will rename tests here and will follow this guideline in the future.

Reply:
@kagg-design tests must be used by developers to understand what the code does and what we are testing.

> Function which name starts with `test` are considered as tests by default

Thanks, but I kind of know that.

What do you think is more readable? `test_get_with_invalid_node` or `it_handles_invalid_nodes`?

> We have tests when function are named in tihs way.

This is either legacy code or simply wrong.

> Anyway, if `it_does` is considered as a company standard, I will rename tests here and will follow this guideline in the future.

Come on Igor. You didn't start yesterday writing code for WPML. You know how we write tests and which convention we use.

If you think we should chance methodology, I'm not against this, but it should be discussed and agreed with the whole team first.

Reply:
@andreasciamanna I have renamed test methods.